shell.nix

```diff
@@ -1,10 +1,18 @@
 { pkgs ? import <nixpkgs> {} }:
 
+with pkgs;
+
 pkgs.mkShell {
   buildInputs = [
-    pkgs.nodejs-15_x
-
-    # keep this line if you use bash
-    pkgs.bashInteractive
+    nodejs-15_x
+    nodePackages.npm
```

Comment:
The node package already comes with npm. If this updates npm to the latest version then I'm on board but otherwise I'd rather omit it unless there's a reason for keeping it.